packages/grid/src/vaadin-grid-keyboard-navigation-mixin.js

```diff
@@ -271,7 +271,7 @@ export const KeyboardNavigationMixin = (superClass) =>
     __isRowExpandable(row) {
```

Comment:
This method is marked as `@private`, but it's already being used in the `A11yMixin`, so I didn't change it. Should it be changed to `@protected`?

Reply:
It's private for external use/extensions. Grid-internal use is fine.